draft-ietf-masque-h3-datagram.md

```diff
@@ -1113,12 +476,7 @@ STREAM(44): HEADERS            -------->
   :method = webtransport
```

Comment:
I think you mean `:protocol = webtransport`, not `:method = webtransport`.  Also, `capsule-protocol = ?1`?

Reply:
Fixed `:protocol`. Regarding `capsule-protocol`, we had decided that one of the examples could go without it to indicate that use of the header is optional